test/e2e/windows/host_process.go

```diff
@@ -154,6 +155,169 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [Excluded:WindowsDoc
 		}
 		framework.ExpectEqual(p.Status.Phase, v1.PodSucceeded)
 	})
+
+	ginkgo.It("container command path validation", func() {
+		tests := []struct {
+			command    []string
+			workingDir string
+		}{
+			{
+				[]string{"cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"c:/Windows/System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"/Windows/System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"c:\\Windows\\System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"System32/cmd.exe", "/ver"}, "c:\\Windows",
+			},
+			{
+				[]string{"System32/cmd.exe", "/ver"}, "c:\\Windows\\",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:\\Windows",
+			},
+			{
+				[]string{"System32/cmd.exe", "/ver"}, "c:/Windows",
+			},
+			{
+				[]string{"System32/cmd.exe", "/ver"}, "c:/Windows/",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:/Windows",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:/Windows\\",
+			},
+			{
+				[]string{"System32\\cmd.exe", "/ver"}, "c:\\Windows/",
+			},
+			{
+				[]string{"%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"%CONTAINER_SANDBOX_MOUNT_POINT%/bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"%CONTAINER_SANDBOX_MOUNT_POINT%\\bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"bin/uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				[]string{"bin\\uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				[]string{"uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
+			},
+			{
+				[]string{"uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin/",
+			},
+			{
+				[]string{"uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\",
+			},
+			{
+				[]string{"powershell", "-c", "cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "c:/Windows/System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "/Windows/System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "c:\\Windows\\System32/cmd.exe", "/ver"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "%CONTAINER_SANDBOX_MOUNT_POINT%\\bin\\uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "%CONTAINER_SANDBOX_MOUNT_POINT%/bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "$env:CONTAINER_SANDBOX_MOUNT_POINT/bin/uname.exe", "-o"}, "",
+			},
+			{
+				[]string{"powershell", "-c", "bin/uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%",
+			},
+			{
+				[]string{"powershell", "-c", "bin/uname.exe", "-o"}, "$env:CONTAINER_SANDBOX_MOUNT_POINT",
+			},
+			{
+				[]string{"powershell", "-c", "uname.exe", "-o"}, "%CONTAINER_SANDBOX_MOUNT_POINT%/bin",
```

Comment:
@dcantah these test cases are failing after I tried them with an hcsshim including https://github.com/microsoft/hcsshim/pull/1137
|  Command |  workingDir  |
|---|---|
| []string{"powershell", "-c", "uname.exe", "-o"} | "%CONTAINER_SANDBOX_MOUNT_POINT%/bin" |
| []string{"powershell", "-c", "uname.exe", "-o"} | "%CONTAINER_SANDBOX_MOUNT_POINT%/bin"/ |
| []string{"powershell", "-c", "uname.exe", "-o"} | "$env:CONTAINER_SANDBOX_MOUNT_POINT\\bin\\

Reply:
these work if you specifying ./uname.exe.
I updated the test cases.